src/lib/sw_engine/tvgSwRaster.cpp

```diff
@@ -596,348 +596,481 @@ static bool _rasterImage(SwSurface* surface, const uint32_t *img, uint32_t w, ui
 /* Gradient                                                             */
 /************************************************************************/
 
-static bool _rasterTranslucentLinearGradientRect(SwSurface* surface, const SwBBox& region, const SwFill* fill)
```

Comment:
i think you don't need to change the function name.  please keep it convention.

Reply:
I changed the name, because I also changed what this func does. I used the same convention as we are using right now. It looks horrible in github, so I'll summarize:

For now we have the external class implementation functions called by the rendering functions. 
Their names are like ```rasterSolidShape``` or ```rasterGradientShape```.

These functions are using the internal class impl functions, like:
- ```rasterSolidShape``` can call (fast track): ```_rasterTranslucentRect``` or ```_rasterSolidRect```
- ```rasterGradientShape``` can call (fast track): ```_rasterTranslucentLinearGradientRect``` and ```_rasterOpaqueLinearGradientRect```

Some of these functions, however, either do the rasterization or choose which function to call - this happens in case of translucent functions. So e.g.:
- ```_rasterTranslucentRect``` chooses between ```_translucentRectAlphaMask```, ```_translucentRectInvAlphaMask``` and ```translucentRect```
- ```_rasterTranslucentLinearGradientRect``` for now handles the cases with the opacity, but the cases with a composition are handled in the ```_rasterOpaqueLinearGradientRect```

This is not as it suppose to be. So after my changes, ```_rasterOpaqueLinearGradientRect``` handles only the cases without the opacity (I mean with op = 255) and without any composition, while ```_rasterTranslucentLinearGradientRect``` chooses between ```_translucentLinearGradientRectAlphaMask```, ```_translucentLinearGradientRectInvAlphaMask``` and ```_translucentLinearGradientRect```. So it's in accordance with the used naming convention